### ISSUE TITLE
Hiding one Column3D series vanishs all others as well

### ISSUE THREAD

**Issue by lticadler** [amcharts/amcharts4#3274]
**Bug description**

Following bug report after #3216 (which works in 4.10.14)
Unfortunately the bug fix in version 4.10.14 made it even worse. Now if there are 2 (or more) Column3D series and you hide one the others vanish as well.

![2021-02-03_05h50_22](https://user-images.githubusercontent.com/58069494/106700244-bc3d1280-65e4-11eb-8435-28d88f6e7c11.gif)

https://codepen.io/chrisadler/pen/ExNVYyW?editors=1010

**Environment (if applicable)**

 - amCharts version: 4.10.14 (not in 4.10.11 and earlier)
 - Browser and its version: all browsers

**Comment by martynasma** [comment]
Fixed in [4.10.15](https://github.com/amcharts/amcharts4/releases/tag/4.10.15).

## [4.10.15] - 2021-02-03

### Fixed
- Fixed issue with toggling multiple `ColumnSeries3D` (introduced in last version).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by lticadler** [comment]
@martynasma Thank you for this really quick fix.
Just to mention: now the Column3D series do no longer animate on new chart data. Don't know, if that was on purpose?
As you can see in the previous pen https://codepen.io/chrisadler/pen/ZEpmLbV the line series is animated but the columns not.

**Comment by martynasma** [comment]
This is a separate issue, which we are not able to solve at this time.

If it's important to you, please do open a separate issue. Thanks and sorry for all the back and forth.
